report_docx/models/ir_actions.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+# © 2018 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models, _
+
+
+class IrActionsReportXml(models.Model):
+    _inherit = 'ir.actions.report.xml'
+
+    ir_values_id = fields.Many2one(
+        comodel_name='ir.values',
+        string='More Menu entry',
+        readonly=True,
+        help='More menu entry.',
+        copy=False
+    )
+
+    report_type = fields.Selection(
+        selection_add=[(('docx', 'Docx'))]
+    )
+
+    template_file = fields.Many2one(
```

Comment:
Can we improve following https://github.com/OCA/reporting-engine/pull/58/files#r69417616?